packages/meditrak-server/src/auth/middleware.js

```diff
@@ -25,6 +26,17 @@ async function authenticateUser(req) {
     return { userId: tokenUserID };
   }
 
+  if (req.sessionCookie?.id) {
+    const sessionId = req.sessionCookie?.id;
+    const sessionModel = new AdminPanelSession(req.database);
+    const session = await sessionModel.findById(sessionId);
```

Comment:
Any reason not to do:
```suggestion
    const session = await req.models.session.findById(sessionId);
```

Reply:
This part of code is no longer relevant!